kubernetes/hands-on-lab/tutorials/deploy.weblogic_short.ocishell.md

```diff
@@ -1,4 +1,4 @@
-# Oracle WebLogic Server Kubernetes Operator Tutorial #
+# WebLogic Kubernetes  Operator Tutorial #
```

Comment:
Another extra space :)

Reply:
You found The TWO places (in the Tutorial and the Contributing doc) that had them. Thank you. Now you are the Eagle Eyes.